helm/kube-prometheus/requirements.yaml

```diff
@@ -1,44 +1,44 @@
 dependencies:
   - name: alertmanager
     version: 0.0.3
-    repository: http://charts.opsgoodness.com
+    repository: https://s3-eu-west-1.amazonaws.com/coreos-charts/stable/
 
   - name: prometheus
     version: 0.0.3
-    repository: http://charts.opsgoodness.com
+    repository: https://s3-eu-west-1.amazonaws.com/coreos-charts/stable/
 
   - name: exporter-kube-api
     version: 0.1.0
-    repository: https://charts.cloudposse.com/incubator
+    repository: https://s3-eu-west-1.amazonaws.com/coreos-charts/stable/
 
   - name: exporter-kube-controller-manager
     version: 0.1.0
-    repository: https://charts.cloudposse.com/incubator
+    repository: https://s3-eu-west-1.amazonaws.com/coreos-charts/stable/
 
   - name: exporter-kube-dns
     version: 0.1.0
-    repository: https://charts.cloudposse.com/incubator
+    repository: https://s3-eu-west-1.amazonaws.com/coreos-charts/stable/
 
   - name: exporter-kube-etcd
     version: 0.1.0
-    repository: https://charts.cloudposse.com/incubator
+    repository: https://s3-eu-west-1.amazonaws.com/coreos-charts/stable/
 
   - name: exporter-kube-scheduler
     version: 0.1.0
-    repository: https://charts.cloudposse.com/incubator
+    repository: https://s3-eu-west-1.amazonaws.com/coreos-charts/stable/
 
   - name: exporter-kube-state
     version: 0.1.2
-    repository: https://charts.cloudposse.com/incubator
+    repository: https://s3-eu-west-1.amazonaws.com/coreos-charts/stable/
 
   - name: exporter-kubelets
     version: 0.1.0
-    repository: https://charts.cloudposse.com/incubator
+    repository: https://s3-eu-west-1.amazonaws.com/coreos-charts/stable/
```

Comment:
I don't see tthe export* in the index

Reply:
I don't get it, what you mean with exporter-*?

Reply:
export-*:  
```
exporter-kubelets
exporter-kube-state
exporter-kube-scheduler
exporter-kube-etcd
...
```
You point them the the s3 bucket.
In the s3 bucket there's the `index.yaml` ? 
in the `index.yaml` generated via the makefile from the PR, those subcharts are not indexed.

Reply:
I moved all the exporter to the root folder, also I added the `make helm-init-repo` to initialize the repo.
Plase run: `make helm-init-repo && make helm-package`